blueocean-dashboard/src/main/js/creation/github/steps/GithubRepositoryStep.jsx

```diff
@@ -15,6 +15,17 @@ export default class GithubRepositoryStep extends React.Component {
         this.props.flowManager.saveSingleRepo();
     }
 
+    _getLoadingMessage() {
+        const { repositoriesLoading } = this.props.flowManager;
+        const count = this.props.flowManager.repositories.length;
+
+        if (repositoriesLoading) {
+            return `Loading Repositories... ${count} so far.`;
```

Comment:
Yep, all I18N work will be done as part of this ticket:  https://issues.jenkins-ci.org/browse/JENKINS-41139